common/scala/src/main/scala/whisk/core/entity/ExecManifest.scala

```diff
@@ -87,7 +89,17 @@ protected[core] object ExecManifest {
         attached: Option[Attached] = None,
         requireMain: Option[Boolean] = None,
         sentinelledLogs: Option[Boolean] = None,
-        image: Option[String] = None)
+        image: Option[String] = None) {
+
+        def toJsonSummary: JsObject = {
+            Map("kind" -> kind.toJson,
+                "deprecated" -> deprecated.getOrElse(false).toJson,
+                "default" -> default.getOrElse(false).toJson,
+                "attached" -> attached.isDefined.toJson,
+                "requireMain" -> requireMain.getOrElse(false).toJson)
```

Comment:
The inner `toJson` are not necessary, either construct the `JsObject` directly and drop the later `toJson.asJsObject` or drop all the inner `toJson`